db/schema.rb

```diff
@@ -10,27 +12,48 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema[7.0].define(version: 2018_02_28_212101) do
+ActiveRecord::Schema[7.0].define(version: 20_221_214_021_307) do
   # These are extensions that must be enabled in order to support this database
-  enable_extension "plpgsql"
+  enable_extension 'plpgsql'
+
+  create_table 'stars', force: :cascade do |t|
+    t.bigint 'user_id', null: false
+    t.bigint 'story_id', null: false
+    t.datetime 'created_at', null: false
+    t.datetime 'updated_at', null: false
```

Comment:
I normally do not edit the schema directly by I use Rubocop which automatically replaces double quotes with single quotes.